CMakeLists.txt

```diff
@@ -244,4 +244,6 @@ if (BUILD_BENCHMARKS)
     add_subdirectory(benchmarks)
 endif ()
 
+add_subdirectory(auxiliary)
```

Comment:
I'm not sure we need the `auxiliary` subdir, could `crash-handler` just not live in the repo root dir?

Reply:
The reason I chose a subdirectory is that I expect other tools to live here in the future. For example, if plugins get some CLI or there's some qt plugin, this could live there.

Reply:
As we discussed, the crashpad project should be a submodule, and it will live under `tools`. Neither of these things need to be actioned until this has been approved